app/pages/home-not-logged-in/discover.jsx

```diff
@@ -53,7 +54,7 @@ const HomePageDiscover = (({ showDialog }) =>
     </div>
 
     <div className="home-discover__image">
-      <img role="presentation" src="/assets/home-computer.png" />
+      <Thumbnail role="presentation" src="//www.zooniverse.org/assets/home-computer.png" width={450} />
```

Comment:
Why did you switch to an absolute url?

Reply:
You can't pass a relative URL to the Thumbnail component.

Reply:
The generated image URL ends up being something like https://thumbnails.zooniverse.org/450x999/www.zooniverse.org/assets/home-computer.png

Although I'm not seeing that for the featured project so I guess thumbnails.zooniverse.org expects the first part to be an actual bucket name on S3.

Reply:
I don't think it's ideal in general to use absolute urls. But /shrug.

Reply:
That was Cam's comment too.

It turns out to  be a bucket path on S3. I'm interested in hearing ideas on how to fix that for staging branches. See my comment above.